### ISSUE TITLE
ReferenceError: mongo is not defined

### ISSUE THREAD

**Issue by zvictor** [kschingiz/meteor-elastic-apm#3]
Unfortunately I couldn't get started. I tried adding `kschingiz:meteor-elastic-apm` to Meteor but I keep getting the error below. I tried running versions 0.0.4, 0.0.3, 0.0.2 and 0.0.1.

```haskell
/.meteor/packages/meteor-tool/.1.7.1-rc.5.1tpqgbc.dswz++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
						throw(ex);
						^

ReferenceError: mongo is not defined
    at Object.ret.(anonymous function) [as update] (packages/kschingiz:meteor-elastic-apm/hijack/db.js:22:38)
    at Collection.update (packages/mongo/collection.js:585:31)
    at Collection.mongoConnectionProto.(anonymous function) [as update] (packages/kschingiz:meteor-elastic-apm/hijack/db.js:50:34)
    at Meteor.startup (server/startup.js:13:28)
    at profileWrapper (/.meteor/local/build/programs/server/profile.js:288:16)
    at Function.time (/.meteor/local/build/programs/server/profile.js:309:28)
    at /.meteor/local/build/programs/server/boot.js:427:13
    at profileWrapper (/.meteor/local/build/programs/server/profile.js:288:16)
    at /.meteor/local/build/programs/server/boot.js:472:5
    at profileWrapper (/.meteor/local/build/programs/server/profile.js:288:16)
    at time (/.meteor/local/build/programs/server/profile.js:309:28)
    at Function.run (/.meteor/local/build/programs/server/profile.js:522:14)
    at /.meteor/local/build/programs/server/boot.js:470:11
=> Exited with code: 1
```

**Comment by kschingiz** [comment]
Yeah, it's my mistake, I will fix it asap.
@zvictor thanks!

**Comment by kschingiz** [comment]
@zvictor Hey, I have published new 0.0.5 version, could you please check it this bug fixed?

**Comment by zvictor** [comment]
the error is gone, thanks!
